movr_stats.py

```diff
@@ -0,0 +1,48 @@
+import numpy
+from tabulate import tabulate
+import time
+
+
+class MovRStats:
+
+    def __init__(self):
+        self.window_stats = {}
```

Comment:
There are going to be a number of threads possibly accessing `self.window_stats` and `self.cumulative_counts` concurrently, so I think all the accesses to them need to be wrapped in a mutex (which shouldn't be too hard since all the direct accesses to it are contained to this class). From what I can tell from a quick google it seems that assignment isn't atomic in python, and `+=` definitely isn't.